sdk/test-utils/test-utils/src/tracing/chaiAzureTrace.ts

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { OperationTracingOptions, useInstrumenter } from "@azure/core-tracing";
+import { assert } from "chai";
+import { MockInstrumenter } from "./mockInstrumenter";
+import { SpanGraph, SpanGraphNode } from "./spanGraphModel";
+
+/**
+ * Augments Chai with support for Azure Tracing functionality
+ *
+ * Sample usage:
+ *
+ * ```ts
+ * import chai from "chai";
+ * import { chaiAzureTrace } from "@azure/test-utils";
+ * chai.use(chaiAzureTrace);
+ *
```

Comment:
TODO: guidelines - what methods must trace, this is how you enforce it with tests, how do we enforce this (linter, code reviews, etc)?